public/plugin_nav.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import {
+  observabilityApplicationsID,
+  observabilityIntegrationsID,
+  observabilityMetricsID,
+  observabilityNotebookID,
+} from '../common/constants/shared';
+import { CoreSetup } from '../../../src/core/public';
+import { AppPluginStartDependencies } from './types';
+import { DEFAULT_NAV_GROUPS, DEFAULT_APP_CATEGORIES } from '../../../src/core/public';
+
+export function registerAllPluginNavGroups(core: CoreSetup<AppPluginStartDependencies>) {
+  core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.observability, [
```

Comment:
Applications is not needed to be added to nav group as it will be deprecated in new left navigation.